cysync/src/store/database/helper.ts

```diff
@@ -1,20 +1,11 @@
 import { ALLCOINS } from '@cypherock/communication';
 
-import logger from '../../utils/logger';
-
-import { priceDb } from './databaseInit';
+import { latestPriceDb } from './databaseInit';
 
 export const getLatestPriceForCoin = async (coin: string) => {
   if (ALLCOINS[coin] && ALLCOINS[coin].isTest) return 0;
 
-  return priceDb.getPrice(coin.toLowerCase(), 7).then(res => {
-    if (res) {
-      const { length } = res.data;
-      return res.data[length - 1][1];
-    }
-    logger.warn(`Cannot find price for coin ${coin}`);
-    return 0;
-  });
+  return (await latestPriceDb.getPrice(coin)).price;
```

Comment:
Keep the previous checks as it is.

Reply:
Got it. Pls check the latest commit @irshadCypherock